jboss/container/eap/galleon/config/mp-elytron/artifacts/opt/jboss/container/eap/galleon/eap-s2i-galleon-pack/src/main/resources/configs/standalone/standalone-microprofile-ha.xml/config.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" ?>
+
+<config xmlns="urn:jboss:galleon:config:1.0" name="standalone-microprofile-ha.xml" model="standalone">
```

Comment:
The Default config doesn't seem correct. That is the EE config adjusted for MP. 
Tthe name of the config is standalone-microprofile-ha.xml. It means that it is merged wit the EAP default standalone-microprofile-ha.xml config.
Some time ago I did an implementation of this image. You will find there the needed adjustments in the default config: https://github.com/jboss-container-images/jboss-eap-modules/commit/00f94d4dc293e380ecf9aff71d588bea851dae50